controllers/mediaController.js

```diff
@@ -0,0 +1,60 @@
+// import Media schema from models
+const { User } = require('../models/mainModels.js');
+// destructured media property of User schema
+const { media } = User;
```

Comment:
As discussed, I believe we won't need line 4 here (we only need the User model from line 2), since we are specifically interfacing with User documents (and the media property of the User document)

Reply:
Seconded